index.html

```diff
@@ -0,0 +1,134 @@
+                            </li>
+                            <li class="item">
+                                <a href="#">Store</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">Mac</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPod</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPhone</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iPad</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">iTunes</a>
+                            </li>
+                            <li class="item">
+                                <a href="#">Support</a>
+                            </li>
+                            <li class="item">
+                                <div class="nav-input">
+                                    <i class="fas fa-search"></i
+                                    ><input class="input-search" type="text" />
+                                </div>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <header class="titles">
+                    <a href="#"><h1>What will your verse be?</h1></a>
+                    <a href="#"
+                        ><h2>
+                            Everyone has something to share. See how people are
+                            using<br />
+                            iPad in their unique ways to add to the world's
+                            story.
+                        </h2></a
+                    >
+                    <a href="#"
+                        ><h3>
+                            Watch the "Your Verse" film and explore the stories
+                        </h3></a
+                    ><a href="#"
+                        ><img
+                            class="play-btn"
+                            src="images/play-icon.png"
+                            alt="play button"
+                    /></a>
+                </header>
+            </main>
+        </div>
+        <div class="flex-container">
+            <div class="flex-item-div">
+                <a href="#"><img src="images/iphone5s.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/iphone5c.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/Mac.jpg" alt="phone" /></a>
+            </div>
+            <div class="flex-item-div">
+                <a href="#"><img src="images/supplier.jpg" alt="phone" /></a>
+            </div>
+        </div>
```

Comment:
**Status: required changes
Hello @emmanuellekamwa** 
Thank you for addressing the raised issues but it seems that some issues are introduced while making the changes. Please fix the following minor issues to get approval for this project. 
1. The four-articles section is overlapping with the main section. one of the images is not visible in [githack](https://raw.githack.com/)
2. left and right footer sections are overlapping. There is no clear gap between them.
Please consider the following image :point_down: 
![Screenshot from 2020-05-25 09-33-02](https://user-images.githubusercontent.com/37747619/82778913-7daa6200-9e72-11ea-9b06-f0dd72914b99.png)